### DISCUSSION TITLE
Routes and redefinition

### DISCUSSION THREAD

**Discussion by viking240285** [munafio/chatify#320] [Q&A]
Hey guys. 
I'm dealing with the chat chatify and its customization https://chatify.munafio.com/customizations .
I need to change and add logic. Now there is a problem - that I can't get to the desired action in the controller via ajax. 
What I did. I published MessagesController (changed the namespace), added a new action to it, added an ajax request to this action in js. Added namespace and in config/chatify.php for routes. 
Added to /routes/web.php a router for a new action, he has decided on the list - I see him. Looks visually to the right place. I call ajax, the request leaves a response of 200. But I definitely don't get into this action. I add the same route to the library /vendor/munafio/chatify/src/routes/web.php the request successfully looks where I need it. 
I've tried a bunch of things, but it doesn't work out correctly to take out robot controls from the library. Redefinition works, adding new actions in the controller is ignored for new routes. 
I will be very grateful to anyone who can lead on the right path.

**Comment by zawizzz** [comment]
idk if you figured out what is going on or not but i had a similar issue and i noticed that there is a problem with routing as some routes are POST routes that fetches data so they should be GET and when you change them to GET the problem you are mentioning occurs
